packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -180,15 +187,15 @@ class CupertinoTextSelectionToolbar extends StatelessWidget {
     );
     final Offset anchorBelowAdjusted = Offset(
       clampDouble(anchorBelow.dx, leftMargin, rightMargin),
-      anchorBelow.dy - _kToolbarContentDistance + paddingAbove,
```

Comment:
Nice!!

Reply:
For the line of code 'anchorBelow.dy - _kToolbarContentDistance + paddingAbove', why is _kToolbarContentDistance subtracted and paddingAbove added? This doesn't seem to have any connection